lib/ansible/module_utils/aws/direct_connect.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+This module adds shared support for Direct Connect modules.
+"""
+
+import traceback
+try:
+    import botocore
+except ImportError:
+    pass
+from ansible.module_utils.ec2 import camel_dict_to_snake_dict
+
+
+class DirectConnectError(Exception):
```

Comment:
Since AnsibleAWSModule is an unstable interface right now this will be a follow up PR.